src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,85 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_WithDataSource_SetsTitleAndDataSource()
```

Comment:
We are using the name format is MethodName_ExpectedBehavior_Condition. So I think it's worth to check if the names are following this format